webapp/app/containers/Widget/components/Chart/Chart.less

```diff
@@ -53,15 +53,20 @@
 }
 
 .table {
-
   &.noBorder {
     :global {
-      .ant-table-thead > tr > th {
-        border: none;
+      .ant-table-thead,
+      .ant-table-thead * {
+        border: none !important;
       }
 
-      .ant-table-tbody > tr > td {
-        border: none;
+      .ant-table-tbody,
+      .ant-table-tbody * {
+        border: none !important;
+      }
+      .ant-table-small,
+      .ant-table-small * {
+        border: none !important;
```

Comment:
Please illustrate why change to this css selector to hit the `noBorder` rule

Reply:
Because the selector does not completely cover the child elements in the table